### DISCUSSION TITLE
How Tailwind is helping to combat Covid-19

### DISCUSSION THREAD

**Discussion by MartinHughes-BPC** [tailwindlabs/tailwindcss#3358] [Kind words]
In the UK all schools and colleges are closed during the current (third) national lockdown. When the college I work for reopens we'll be carrying out regular Covid-19 testing for students and staff.

To that end, this week I've been creating a Laravel (with some Vue2 interactivity) application so we can efficiently record test results and perform track and trace for those who may have been in contact with positive cases. I naturally reached for Tailwind and TailwindUI for all my styling - and even though I'm not the greatest graphic designer in the world (or programmer, for that matter!), I have something that I think looks more than reasonable in double-quick time!

I wish I could actually show you guys something, but I'm essentially dealing with lots of confidential live data rather than test data (I have to integrate with a 3rd party MIS database) - but I wanted to say thanks for Tailwind, it's made my life a lot easier!

**Comment by simonswiss** [comment]
Hey @MartinHughes-BPC, thanks for sharing this! ❤️

Very sad to see the state of things in the UK. I myself have an extended family member in London who is in critical condition in ICU...

Hoping things take a turn for the better soon. Looking forward to seeing how your project looks (if it's ever publicly available), and happy to head Tailwind & Tailwind UI is helping you get where you need in record times 👍
